modules/ptp-dual-ports-oc.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/about-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ptp-dual-ports-oc_{context}"]
+= Using dual-port NICs to improve redundancy for PTP ordinary clocks
+
+{product-title} supports single and dual-port networking interface cards (NIC) as ordinary clocks for PTP timing. To improve redundancy, you can configure a dual-port NIC with one port as active and the other as standby.
+
+:FeatureName: Configuring linuxptp services as an ordinary clock with dual-port NIC redundancy
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+In this configuration, the ports in a dual-port NIC operate as follows:
+
+* The active port functions as an ordinary clock in the `Following` port state.
```

Comment:
If these actions are in a sequence, user numbering as against bullet points. Feel free to ignore this comment if the sequence does not apply.

Reply:
I think bullets work better here as ports dynamically change back and forth